README.md

```diff
@@ -84,6 +84,11 @@ List {
 .introspectTableView { tableView in
     tableView.separatorStyle = .none
 }
+.introspectTableViewCell { cell in
```

Comment:
Same, might remove if we agree on above.

Reply:
See my comment above